plugin.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "Database Items",
-    "description": "Provide database access via items.",
+    "name": "Database Assetstore",
```

Comment:
We should change Item Previews, I'd say we should keep Google Analytics capitalized since it's proper.